docs/versioning.md

```diff
@@ -0,0 +1,7 @@
+# Versioning
+
+Argo Workflows does not uses Semantic Versioning yet. This is because we have not introduced any breaking changes since v2.0.
```

Comment:
```suggestion
Argo Workflows does not use Semantic Versioning yet. This is because we have not introduced any breaking changes since v2.0.
```

I don't understand this. If we haven't introduced breaking changes since 2.0, doesn't this mean that we _actually_ use SemVer?

Reply:
I have a feeling that we're saying - in the past we have been SemVer, but we won't be in the future.

Reply:
Since when? I thought we were set to be SemVer.

Regardless I think that statement is contradictory: not introducing breaking changes since 2.0 is using SemVer if we're currently on v2.x.

> in the past we have been SemVer, but we won't be in the future.

Unclear to me how its saying that.